docs/table/mixin_columns.rst

```diff
@@ -202,7 +223,10 @@ that contain mixin columns:
 **Mixin column attributes**
 
 For mixin columns the column attributes ``name``, ``unit``, ``dtype``,
-``format``, ``description`` and ``meta`` are currently stored in a simple
+``format``, ``description`` and ``meta`` are stored in the ``info`` attribute
+
+
+
 dictionary called `_astropy_column_attrs`.  These attributes can be manipulated
```

Comment:
I think this is no longer a "caveat".  The interface is now fully described in the section on Mixin Attributes, so I'll just delete this whole sub-section here.